packages/application-shell/src/components/application-shell-authenticated/new-main-navigation-flag-wrapper.tsx

```diff
@@ -14,10 +15,9 @@ const NewMainNavigationFlagWrapper = (props: {
   }: TNewMainNavigationFlagWrapperChildrenProps) => JSX.Element;
 }) => {
   const isNewNavigationEnabled = useFeatureToggle(MAIN_NAVIGATION);
```

Comment:
@tdeekens I have one question. Now the value of `isNewNavigationEnabled` flag is always false. And also there's no network call to Darkly like before the changes.

But maybe I'm missing something perhaps? 
The same is currently on main before since [your last](https://github.com/commercetools/merchant-center-application-kit/pull/3001) changes(I think).

Reply:
The network call before the changes ⬇️ 
<img width="1430" alt="Screenshot 2023-08-21 at 11 09 24" src="https://github.com/commercetools/merchant-center-application-kit/assets/52276952/47cd7869-b087-4138-80ff-a2a001f30d68">